### ISSUE TITLE
Cleanup design on transcription page to bring into compliance w/LC standards

### ISSUE THREAD

**Issue by lalgee** [LibraryOfCongress/concordia#1404]
Cleanup design on transcription page to bring into compliance w/LC standards.

@klee72 commented on https://github.com/LibraryOfCongress/concordia/issues/1340#issuecomment-826975313:

Currently there are many css variables (example: var(--primary-color)) being used on Concordia, especially for colors. This is because the project has been using the precompiled Bootstrap css where we couldn't customize the theme from more deeper level. LC has recently switched over to SASS version of Bootstrap, and some colors have been replaced into SASS variables. However CSS has not yet cleaned up transcription detail page yet.

**Comment by lalgee** [comment]
The changes all look good to me here -- @elainekamlley what should the process be for testing this one?

**Comment by klee72** [comment]
@lalgee @elainekamlley @Igor-Ino @jkue
This is a list of CSS styles that are either replaced or removed:
[README.md](https://github.com/LibraryOfCongress/concordia/files/6467115/README.md)

I think a sanity check throughout the app would be a good idea, just to see if there's anything that's affected negatively by the color update. And although this update shouldn't have affected the functionalities of transcription page, but should be tested to see if everything is working ok.

**Comment by lalgee** [comment]
Ok, sounds good to commit. Let us know when it's available to test. @klee72

**Comment by klee72** [comment]
@lalgee @elainekamlley @Igor-Ino @jkue
Updates have been pushed to dev. Can you guys test on dev? Otherwise @rabiloc is out until Friday so it can be pushed to test on Friday, unless @jkueloc can push it to test.

**Comment by lalgee** [comment]
We can test in dev @klee72. Probably for the best since I just documented a bug in test.

**Comment by lalgee** [comment]
All looks good in dev!
